one/src/daemon.rs

```diff
@@ -236,6 +237,33 @@ pub struct DaemonOpts {
         env = "CERAMIC_ONE_ETHEREUM_RPC_URLS"
     )]
     ethereum_rpc_urls: Vec<String>,
+
+    /// Enable the aggregator, requires Flight SQL to be enabled
+    #[arg(
+        long,
+        requires = "flight_sql_bind_address",
+        env = "CERAMIC_ONE_AGGREGATOR"
+    )]
+    aggregator: Option<bool>,
+
+    /// S3 bucket name.
+    /// When configured the aggregator will support storing data in S3 compatible object stores.
+    ///
+    /// Credentials are read from the environment:
+    ///
+    ///   * AWS_ACCESS_KEY_ID -> access_key_id
+    ///   * AWS_SECRET_ACCESS_KEY -> secret_access_key
+    ///   * AWS_DEFAULT_REGION -> region
+    ///   * AWS_ENDPOINT -> endpoint
+    ///   * AWS_SESSION_TOKEN -> token
+    ///   * AWS_ALLOW_HTTP -> set to "true" to permit HTTP connections without TLS
+    ///
+    #[arg(
+        long,
+        requires = "experimental_features",
+        env = "CERAMIC_ONE_S3_BUCKET"
```

Comment:
in the future the data files might live in a different bucket than the aggregator output right?  So we might want this name to indicate that this specifically for the result of the aggregated state that we are exporting.

Reply:
I am anticipating a single bucket for all the data. A single bucket and handle many tables. I don't want to complicate the API by expecting a different bucket for each table until that need arises in product discussions.

Reply:
I'll update the comment to reflect this intention